app/streaming/session.cpp

```diff
@@ -275,6 +275,16 @@ int Session::getDecoderCapabilities(StreamingPreferences::VideoDecoderSelection
     return caps;
 }
 
+NvComputer *Session::getComputer() const
+{
+    return m_Computer;
+}
+
+bool Session::shouldQuitAppAfter() const
+{
+    return m_Preferences->quitAppAfter;
```

Comment:
This should be exposed via StreamingPreferences itself, not the Session class.

Reply:
You mean that Session would expose the whole StreamingPreferences object? Not just this single option?

Reply:
Note that I can't just use any StreamingPreferences object. I must be the one that is associated with the session as it may have values overwritten from command line.